### ISSUE TITLE
Grammar suggestions for footnotes

### ISSUE THREAD

**Issue by timwaterhouse** [metrumresearchgroup/pmparams#82]
Can I please ask @KatherineKayMRG to weigh in on these suggestions for updates to `param_notes()`?

- `ci`: change "intervals" to "interval"
- `cvOmegaEq`: change "omegas" to "omega"
- `logTrans`: change "log-domain" to "log domain"
- `logitTrans`: change "logit-domain" to "logit domain"

(current list is [here](https://github.com/metrumresearchgroup/pmparams/blob/b7183004414dfaadb096f6bed425077a22c99750/R/param-notes.R#L59-L69) for reference)

**Comment by KatherineKayMRG** [comment]
@timwaterhouse I agree with all those updates. It would make them more consistent with other footnotes
